lib/logstasher.rb

```diff
@@ -57,7 +58,7 @@ def log_as_json(payload, as_logstash_event: false)
                        payload.to_json
                      end
 
-      logger << json_payload + $INPUT_RECORD_SEPARATOR
+      logger << (prefix.nil? ? "" : prefix) + json_payload + $INPUT_RECORD_SEPARATOR
```

Comment:
`prefix.to_s` will work here since `nil.to_s #=> ""`.

Reply:
This might actually be a big problematic since the expectation is that the output is a pure json blob without any formatting or parsing required.

Reply:
Won't this break logstash filters that try to parse a json payload?

```
    filter {
      json {
        source => "message"
      }
    }
```